extensions/visualization/README.md

```diff
@@ -0,0 +1,76 @@
+# Visualization Parameter Specification (`visualization`)
+
+## Item Fields
+
+| Field Name                    | Type       | Description                                                                                     |
+|-------------------------------|------------|-------------------------------------------------------------------------------------------------|
+| vis:display_name              | string     | Name of the displayed visualization.                                                            |
+| vis:vis                       | \*         | Parameters for visualizing the dataset as an image.                                             |
+| vis:lookat                    | LatLonZoom | Coordinate on which the visualization is initially centered.                                    |
+| vis:filter                    | Filter     | Optional structure translating to a query into the dataset.                                     |
+| vis:preview_image_script_name | string     | Optional URL linking to the JavaScript script used to create the sample/thumbnail image.        |
+| vis:earth_engine_script_name  | string     | Optional URL linking to the JavaScript script file used for the Earth Engine site code snippet. |
+
+\* = One of the Image Visualization, Table Visualization, or Polygon Visualization, defined below:
+
+### Image Visualization
+
+| Field Name | Type    | Description                                            |
+|------------|---------|--------------------------------------------------------|
+| band_vis   | BandVis | Vis args for the given band_names.                     |
+| global_vis | BandVis | If vis_args are present here, they apply to all bands. |
```

Comment:
It looks like there are two places where you can specify global viz arguments -- in `band_vis`, if you provide only one value in the array parameters, or here -- am I reading this correctly that those could conflict? If so, does `global_vis` take precedence? Under what circumstances should someone provide both `global_vis` and `band_vis`?

Reply:
In Earth Engine it's handled like this:
In `ImageVisualization` a number of `band_names` are specified. For each of these bands, the BandVis in `band_vis` is applied. But for ALL bands of the Image/ImageCollection, the visualization parameters in `global_vis` are applied.